charts/kubefed/charts/controllermanager/templates/clusterrolebindings.yaml

```diff
@@ -17,7 +17,14 @@ subjects:
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
-  name: auth-delegator-kubefed-admission-webhook
+# For namespace scoped deployments, create a unique cluster-scoped resource
+# name using the namespace. This is needed because admission-webhooks do not
+# currently support the ability to have namespace-scoped RBAC permissions only.
+{{- if and .Values.global.scope (eq .Values.global.scope "Namespaced") }}
+  name: kubefed-admission-webhook:{{ .Release.Namespace }}:auth-delegator
+{{ else }}
+  name: kubefed-admission-webhook:auth-delegator
```

Comment:
Is there a reason not to qualify the name of the cluster-scoped delegator with the system namespace, so that the name form would be consistent across deployment modes?  This comment applies to other instances of namespace-qualifying only the resources created for a namespaced deployment rather than namespace-qualifying all resources created for either type of deployment.

Reply:
It certainly crossed my mind as it would have been easier, but then it wouldn't be consistent with the other non-webhook cluster-scoped resource names. Also, having the same cluster-scoped resource names helps prevent duplicate cluster-scoped kubefed deployments.

Reply:
I don't think the latter argument is worth worrying about - if we're to avoid duplicate deployments lets make it explicit.

Regarding naming consistency, afaik we already use consistent naming for some resources and maybe we should use that as the norm?  cc: @xunpan

Reply:
Since we do not currently support explicitly avoiding duplicate deployments, I think blocking it implicitly by way of duplicate resource names inherent in the design of our helm chart, along with some documentation that multiple cluster-scoped deployments should be avoided altogether, seems like a reasonable alternative.

In any case, if we want to change all the cluster-scoped resource names, let's open up a new PR to address it.